src/reader-activation-auth/accessibility.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Trap focus in the modal when opened.
+ * See: https://uxdesign.cc/how-to-trap-focus-inside-modal-to-make-it-ada-compliant-6a50f9a70700
+ */
+export function trapFocus( currentModal ) {
+	const focusableEls =
+		'button, [href], input:not([type="hidden"]), select, textarea, [tabindex]:not([tabindex="-1"])';
+	const visibleFocusableEls = [];
+
+	const firstFocusableEl = currentModal.querySelectorAll( focusableEls )[ 0 ]; // get first element to be focused inside modal
```

Comment:
We probably won't run into this case since we control the markup of our modals, but generally, we'd want to confirm the length of this query before accessing one of its elements.

```suggestion
	const firstFocusableEl = currentModal.querySelectorAll( focusableEls )?.[ 0 ]; // get first element to be focused inside modal
```

Reply:
Thanks Raz! This has been updated!